src/scss/app.scss

```diff
@@ -1,13 +1,122 @@
+@import "utilities/media";
+@import "utilities/typography";
+@import "utilities/normalise";
 
-$primary-color: green;
+$background-primary: #36393f;
+$background-secondary: #2f3136;
+$background-secondary-alt: #292b2f;
+$background-tertiary: #202225;
+$background-accent: #4f545c;
+$background-floating: #18191c;
+$border-primary: rgba(4,4,5,0.5);
+$interact-hover: #dcddde;
+$interact-active: #fff;
+$interact-muted: #4f545c;
```

Comment:
Would be good to have a chat about how you can name colour variables so that they're declarative. I'll accept this for now, and send you a pm